### PR TITLE
Prepare 3.4.4 release

### PR DESCRIPTION
**Pull request by ivan-mogilko**
By [former request](https://github.com/adventuregamestudio/ags/issues/785#issuecomment-495952217) (although delayed more than I planned), this prepares a 3.4.4 update, which only contains critical bugfixes and compatibility fixes to be added on top of 3.4.*.

Notable fixes:
* fixed IsMusicVoxAvailable() in old games that had music.vox;
* fixed Character.GetDestinationY not working if coordinate is >255;
* fixed blinking portrait animations in old 32-bit games;
* fixed some very old games (like Ben Jordan 3) not displaying correct gfx on default inventory screen buttons.
* support loading experimental 3.3.0 CR games ("Skumring").
* allow to link to both Allegro 4.4.2 and 4.4.3.

### CODE REVIEW

**Comment by morganwillcock** [comment]
Can this also include https://github.com/adventuregamestudio/ags/commit/db08123e5a407bcd400ed19361a9646e745cdcc5 please.

**Comment by ivan-mogilko** [comment]
> Can this also include db08123 please.

I tried and this line causes error:
`ServicePointManager.SecurityProtocol = SecurityProtocolType.Tls | SecurityProtocolType.Tls11 | SecurityProtocolType.Tls12;`

Tls11 and Tls12 are not part of SecurityProtocolType. That version still uses .NET 2.0. Is it okay to remove these last two?

**Comment by morganwillcock** [comment]
Maybe try casting them? They would be `(SecurityProtocolType)768` and `(SecurityProtocolType)3072` respectively, but I don't know what the runtime environment will do. The server still isn't working (from what I can tell) but I think as long as you don't get an SSL error on crash submission it is probably fine.

**Comment by ivan-mogilko** [comment]
>  The server still isn't working (from what I can tell) but I think as long as you don't get an SSL error on crash submission it is probably fine.

Maybe I am missing something, but  I am getting exactly same error message in either case, saying "Connection was closed unexpectedly". I tried `ServicePointManager.SecurityProtocol = SecurityProtocolType.Ssl3`, `ServicePointManager.SecurityProtocol = SecurityProtocolType.Tls` and without this line at all.

**Comment by morganwillcock** [comment]
It sounds like it just cuts you off before anything is intiated. Maybe this is regional firewalling, or Apache security being heavy handed, or maybe the server side is just more broken than when I tried it a couple of days ago. When I tested the 3.5.0 branch I got an internal server error (500) so the SSL part was working. I'll see if I can try test it again with the 3.4.4 branch.

**Comment by morganwillcock** [comment]
I tried your branch as-is and got the connection closed error, then I added these changes:
<pre>diff --git a/Editor/AGS.Editor/AGSEditor.cs b/Editor/AGS.Editor/AGSEditor.cs
index 64c429cc..6dac24ad 100644
--- a/Editor/AGS.Editor/AGSEditor.cs
+++ b/Editor/AGS.Editor/AGSEditor.cs
@@ -10,6 +10,7 @@ using System.Xml;
 using AGS.CScript.Compiler;
 using AGS.Types;
 using AGS.Types.Interfaces;
+using System.Net;
 
 namespace AGS.Editor
 {
@@ -239,6 +240,9 @@ namespace AGS.Editor
 
         public void DoEditorInitialization()
         {
+            // disable SSL v3
+            ServicePointManager.SecurityProtocol = SecurityProtocolType.Tls | (SecurityProtocolType)768 | (SecurityProtocolType)3072;
+
             _game = new Game();
             _sourceControl = new SourceControlProvider();
             _recentGamesList = new RecentGamesList();</pre>

At this point I got the error reponse from the server (so it defintely connected to it). I think these changes probably fix the underlying SSL issue and .NET 2.0 probably just returns a less helpful error message that doesn't mention SSL, or maybe it just fails earlier.

To sum up, I think this change and the casting of SecurityProtocolType is OK.
